app/services/permissions/user.rb

```diff
@@ -0,0 +1,30 @@
+module Permissions
+  module User
+    extend ActiveSupport::Concern
+
+    def can_edit?(org)
+      superadmin? || (!org.nil? && organisation == org)
+    end
+
+    def can_delete?
+      superadmin?
```

Comment:
@tansaku is `superadmin?` scoped to an organisation? If so this could lead to a security issue and a user can delete any organisation in the system.

Reply:
No superadmin is not scoped to a particular organisation - it's literally the super-admin who can do anything